build_mesos

```diff
@@ -159,7 +159,13 @@ function build {(
   ./bootstrap
   mkdir -p "$build_dir"
   cd "$build_dir"
-  "$src_dir"/configure $(configure_opts)
+
+  # A bug in Autoconf 2.64 and earlier causes `AC_CONFIG_LINKS(src/a/b:src/c/d)`
+  # to create symlink 'src/a/b' that points to 'src/c/d' instead of pointing to
+  # '../../src/c/d'. This bug is triggered only when invoking configure script
+  # with an absolute path. Thus, we use relative invocation here.
+  # http://git.savannah.gnu.org/cgit/autoconf.git/commit/?id=13e3570
+  ../configure $(configure_opts)
```

Comment:
Removing `$src_dir` puts this change at greater risk of breaking existing behavior. Would need to prove that this still builds properly when `--src-dir` is specified and when it is not. Otherwise, suggest finding a way to convert `$src_dir/configure` to a path relative to `$build_dir`.